torch_xla/torch_xla.py

```diff
@@ -59,20 +59,14 @@ def device_count() -> int:
   return len(real_devices())
 
 
-def sync(wait: bool = False):
+def sync(wait: bool = False, reset_scope: bool = True):
   """Launches all pending graph operations.
 
   Args:
     wait (bool): whether to block the current process until the execution finished.
-
+    reset_scope (bool): whether to reset the tracing scope of lazy tensor.
```

Comment:
Could you elaborate what this means? By reading the code, I wasn't sure what does "resetting" mean. Does it mean any tracing scope set by the user for the profiler are invalidated? Maybe could you run some simple tests to verify its behavior?

Reply:
This is addressed in #9086 with more detail on where the flag came from.